src/api/spec/models/package_spec.rb

```diff
@@ -440,12 +440,22 @@
       before { stub_request(:post, backend_url) }
 
       it { is_expected.to be_truthy }
+
+      it "without error" do
```

Comment:
because if you don't execute the `subject` the project won't have any error :confused: